src/main/java/com/google/devtools/build/lib/rules/android/AndroidConfiguration.java

```diff
@@ -881,6 +881,14 @@ public static AndroidManifestMerger fromString(String value) {
         help = "Use R.txt from the merging action, instead of from the validation action.")
     public boolean useRTxtFromMergedResources;
 
+    @Option(
+            name = "output_library_linked_resources",
+            defaultValue = "true",
+            documentationCategory = OptionDocumentationCategory.UNDOCUMENTED,
+            effectTags = {OptionEffectTag.UNKNOWN},
+            help = "If disabled, does not provide library.ap_ outputs for library targets")
```

Comment:
Could you also include the PR description here? The PR description has great context, e.g. reducing download overhead and devices not requiring these files.

Reply:
Done - thanks for the review!